### ISSUE TITLE
%type<t>XXX after %token XXX not be recognized

### ISSUE THREAD

**Issue by RussellGuo** [babyraging/yash#33]
Hello,

Sorry for brothering you. As the title, I declared a token by %token XXX, then I defined it's type ‘t’ by %type<t> XXX. After that a warning 'Token declared but never used.' above the XXX appears. I think the plugin recognizes %type as a non-terminal definition. That syntax is allowed by bison (even in YACC I guess) and can be used.

thanks a lot.

**Comment by vikram18rock** [comment]
I agree with @RussellGuo. pasting a screenshot for clarity on the issue
![image](https://github.com/babyraging/yash/assets/100817414/f128f51c-4de5-407b-8402-05f6cdef37f1)
